frontend/src/reducers/rootReducer.tsx

```diff
@@ -2,10 +2,19 @@ import { combineReducers } from 'redux';
 import parcelsReducer from 'reducers/parcelsReducer';
 import lookupCodeReducer from 'reducers/lookupCodeReducer';
 import * as reducerTypes from 'constants/reducerTypes';
+import networkReducer from "./networkReducer";
```

Comment:
We should use single quotes here as per our JS styleguide. This should be auto-fixed by the linter thought...